### ISSUE TITLE
warning " > @cypress/webpack-dev-server@1.1.1" has incorrect peer dependency "webpack-dev-server@>=4".

### ISSUE THREAD

**Issue by DamienCassou** [cypress-io/cypress#15805]
`@cypress/webpack-dev-server` has a `peerDependency` on `"webpack-dev-server": ">=4"` but no such version exist: the most recent is currently 4.0.0-beta.1.

This results in warnings during installation:

```js
warning " > @cypress/webpack-dev-server@1.1.1" has incorrect peer dependency "webpack-dev-server@>=4".
```

References:
- https://www.npmjs.com/package/webpack-dev-server
- https://github.com/cypress-io/cypress/blob/master/npm/webpack-dev-server/package.json#L30

**Comment by JessicaSachs** [comment]
@DamienCassou we released `@cypress/webpack-dev-server` [v1.1.2](https://www.npmjs.com/package/@cypress/webpack-dev-server) which fixes the package.json to specify the right version. Thanks for the catch.

**Comment by DamienCassou** [comment]
Thank you @agg23 and @JessicaSachs.
